spec_linux.go

```diff
@@ -0,0 +1,148 @@
+// +build linux
+
+package specs
+
+// LinuxSpec is the full specification for linux containers.
+type LinuxSpec struct {
+	Spec
+	// Linux is platform specific configuration for linux based containers.
+	Linux Linux `json:"linux"`
+}
+
+// Linux contains platform specific configuration for linux based containers.
+type Linux struct {
+	// UidMapping specifies user mappings for supporting user namespaces on linux.
+	UidMappings []IDMapping `json:"uidMappings"`
+	// UidMapping specifies group mappings for supporting user namespaces on linux.
+	GidMappings []IDMapping `json:"gidMappings"`
+	// Rlimits specifies rlimit options to apply to the container's process.
+	Rlimits []Rlimit `json:"rlimits"`
+	// SystemProperties are a set of key value pairs that are set for the container on start.
+	SystemProperties map[string]string `json:"systemProperties"`
```

Comment:
If we get in https://github.com/opencontainers/runc/pull/73 then might save the rename to sysctl after ;)
I can rebase later however :)